doc/sphinx/Pacemaker_Explained/operations.rst

```diff
@@ -659,6 +659,7 @@ path.
 
 Also, if a migratable resource depends on a non-migratable resource, and both
 need to be moved, the migratable resource will be restarted.
+
```

Comment:
Can you kick out this change in its own PR real quick and backport to 2.1? Worth getting in 2.1.8

Reply:
#3595 
#3596